src/components/MainPage.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable react/jsx-closing-tag-location */
+/* eslint-disable object-curly-newline */
+/* eslint-disable react/jsx-one-expression-per-line */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
+import { Link } from 'react-router-dom';
+// import Swiper for carrousel
```

Comment:
- [ ] Please, kindly remove this old piece of commented code, to improve the cleanliness of the repo 💪